pkg/clusterregistry/server.go

```diff
@@ -57,7 +57,7 @@ func NonBlockingRun(s *options.ServerRunOptions, stopCh <-chan struct{}) error {
 	if err := s.SecureServing.MaybeDefaultWithSelfSignedCerts(s.GenericServerRunOptions.AdvertiseAddress.String(), nil, nil); err != nil {
 		return fmt.Errorf("error creating self-signed certificates: %v", err)
 	}
-	s.SecureServing.ForceLoopbackConfigUsage()
+	//s.SecureServing.ForceLoopbackConfigUsage()
```

Comment:
Why?

Reply:
This method was removed from `SecureServing`. I forgot to remove the commented-out code!